pinot-core/src/main/java/org/apache/pinot/core/common/datatable/DataTableImplV2.java

```diff
@@ -50,51 +46,19 @@
   // VARIABLE_SIZE_DATA (START|SIZE)
   private static final int HEADER_SIZE = Integer.BYTES * 13;
 
-  private final int _numRows;
-  private final int _numColumns;
-  private final DataSchema _dataSchema;
-  private final int[] _columnOffsets;
-  private final int _rowSizeInBytes;
-  private final Map<String, Map<Integer, String>> _dictionaryMap;
-  private final byte[] _fixedSizeDataBytes;
-  private final ByteBuffer _fixedSizeData;
-  private final byte[] _variableSizeDataBytes;
-  private final ByteBuffer _variableSizeData;
-  private final Map<String, String> _metadata;
-
   /**
    * Construct data table with results. (Server side)
    */
   public DataTableImplV2(int numRows, DataSchema dataSchema, Map<String, Map<Integer, String>> dictionaryMap,
       byte[] fixedSizeDataBytes, byte[] variableSizeDataBytes) {
-    _numRows = numRows;
-    _numColumns = dataSchema.size();
-    _dataSchema = dataSchema;
-    _columnOffsets = new int[_numColumns];
-    _rowSizeInBytes = DataTableUtils.computeColumnOffsets(dataSchema, _columnOffsets);
-    _dictionaryMap = dictionaryMap;
-    _fixedSizeDataBytes = fixedSizeDataBytes;
-    _fixedSizeData = ByteBuffer.wrap(fixedSizeDataBytes);
-    _variableSizeDataBytes = variableSizeDataBytes;
-    _variableSizeData = ByteBuffer.wrap(variableSizeDataBytes);
-    _metadata = new HashMap<>();
+    super(numRows, dataSchema, dictionaryMap, fixedSizeDataBytes, variableSizeDataBytes);
   }
 
   /**
    * Construct empty data table. (Server side)
    */
   public DataTableImplV2() {
-    _numRows = 0;
-    _numColumns = 0;
-    _dataSchema = null;
-    _columnOffsets = null;
-    _rowSizeInBytes = 0;
-    _dictionaryMap = null;
-    _fixedSizeDataBytes = null;
-    _fixedSizeData = null;
-    _variableSizeDataBytes = null;
-    _variableSizeData = null;
-    _metadata = new HashMap<>();
+    super();
```

Comment:
This constructor can be removed. super() is redundant.

Reply:
Done. `super()` is redundant, but default constructor is needed here since we have a `DataTableImplV2(ByteBuffer byteBuffer)` no-default constructor.